pkg/agent/route/interfaces.go

```diff
@@ -51,6 +52,24 @@ type Interface interface {
 	// DeleteSNATRule should delete rule to SNAT outgoing traffic with the mark.
 	DeleteSNATRule(mark uint32) error
 
+	// InitService should add the basic TC configuration on Linux.
+	InitService(nodePortIPMap map[int][]net.IP, isIPv6 bool) error
```

Comment:
The name InitService is too generic. If it is for NodePort only, call it InitNodePortRoutes; if it also handles other types, call it InitServiceProxyRoutes.

Reply:
This func will do:
- Remove stale LoadBalancer/ClusterIP route entries
- Add Linux basic filters for NodePort/LoadBalancer
- Add route entry for routing packets whose destination IP is serviceGWHairpinIP to Antrea gateway.

Maybe call it InitServiceProxyConfig?

Reply:
InitServiceProxyConfig sounds good to me.